### PR TITLE
namespace gc count result num

### PR DESCRIPTION
**Pull request by kakj-go**
#### What type of this PR

/kind bug

#### What this PR does / why we need it:
namespace gc count result num not sql result total

### CODE REVIEW

File: modules/pipeline/pipengine/reconciler/pipeline_gc_compensator.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func (r *Reconciler) doWaitGCCompensate(isSnippetPipeline bool) {
 	}
 }
 
-func (r *Reconciler) getNeedGCPipelines(pageNum int, isSnippet bool) ([]spec.Pipeline, int64, error) {
+func (r *Reconciler) getNeedGCPipelines(pageNum int, isSnippet bool) ([]spec.Pipeline, int, error) {
```
Comment (Effet) [inline modules/pipeline/pipengine/reconciler/pipeline_gc_compensator.go:63]: explain why int64 -> int

File: modules/pipeline/pipengine/reconciler/pipeline_gc_compensator.go [hunk 2]
```diff
@@ -72,7 +72,7 @@ func (r *Reconciler) getNeedGCPipelines(pageNum int, isSnippet bool) ([]spec.Pip
 	}
 	req.AllSources = true
 	req.IncludeSnippet = isSnippet
-	pipelines, _, total, _, err := r.dbClient.PageListPipelines(req)
+	pipelines, _, _, _, err := r.dbClient.PageListPipelines(req)
```
Comment (Effet) [inline modules/pipeline/pipengine/reconciler/pipeline_gc_compensator.go:75]: PageListPipelines has too many returns, should refactor it

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/pipeline/pipengine/reconciler/pipeline_gc_compensator.go (hunk 1 above)
Comment (kakj-go) [reply modules/pipeline/pipengine/reconciler/pipeline_gc_compensator.go:63]: result `len(pipelines)` was int type

File: modules/pipeline/pipengine/reconciler/pipeline_gc_compensator.go (hunk 2 above)
Comment (kakj-go) [reply modules/pipeline/pipengine/reconciler/pipeline_gc_compensator.go:75]: yes, but not now

**Review by sfwn — APPROVED** [review]

**Pull request merged**